src/operator/main.go

```diff
@@ -135,9 +140,9 @@ func main() {
 		logrus.WithError(err).Fatal(err, "unable to start manager")
 	}
 
-	kafkaServersStore := kafkaacls.NewServersStore(tlsSource, enableKafkaACLCreation, kafkaacls.NewKafkaIntentsAdmin)
+	kafkaServersStore := kafkaacls.NewServersStore(tlsSource, enableKafkaACLCreation, kafkaacls.NewKafkaIntentsAdmin, enforcementEnabledGlobally)
```

Comment:
This is a matter of taste and simplicity, so putting this here as a thought and let's discuss if you disagree:

IMO it would have been simpler (and clear enough) to keep the `enforcementEnabledGlobally` flag only here on main (and probably in the component that's going to report this status upwards to the cloud), and have the specific reconcilers use their dedicated flags without having to pass the `enforcementEnabledGlobally` around. Meaning: 
```
kafkaServersStore := kafkaacls.NewServersStore(tlsSource, enableKafkaACLCreation && enforcementEnabledGlobally, kafkaacls.NewKafkaIntentsAdmin)
...
endpointReconciler := external_traffic.NewEndpointsReconciler(mgr.GetClient(), mgr.GetScheme(), enforcementEnabledGlobally && autoCreateNetworkPoliciesForExternalTraffic)
... 
```
Another advantage of this approach is that it's safer, in that you don't have to make sure you propagate and support this global flag in all existing and future reconcilers. Every reconcile should just have an on/off switch, and the global disable flag just disables all switches.